pkg/expression/distsql_builtin.go

```diff
@@ -581,8 +581,6 @@ func getSignatureByPB(ctx BuildContext, sigCode tipb.ScalarFuncSig, tp *tipb.Fie
 		f = &builtinCompressSig{base}
 	case tipb.ScalarFuncSig_MD5:
 		f = &builtinMD5Sig{base}
-	case tipb.ScalarFuncSig_Password:
-		f = &builtinPasswordSig{base}
```

Comment:
Does this mean we should clean up the protobuf/tikv/tiflash repos as well? Maybe just create deprecation issues in their repositories?

Reply:
I would expect this to be listed on one of these pages:
- https://docs.pingcap.com/tidb/stable/expressions-pushed-down
- https://docs.pingcap.com/tidb/stable/tiflash-supported-pushdown-calculations

But I don't see it there. 

Looking at the sourcecode I see this:
- https://github.com/tikv/tikv/blob/7ef4aa9549f136cabdac7e5177f39fe16631b514/components/tidb_query_expr/src/impl_encryption.rs#L141-L161
- https://github.com/pingcap/tiflash/blob/ea3620547107437a9ead0ed7b7075bcf7a08d474/dbms/src/Flash/Coprocessor/DAGUtils.cpp#L407

So it looks cleanup is only needed for TiKV.
https://github.com/tikv/tikv/issues/18031

Reply:
We *could* cleanup https://github.com/pingcap/tipb/blob/8408c760e822f7cd7564ce280842af8a417f0a47/proto/expression.proto#L405 but I think it might be better to leave this.